web/beacon-app/src/constants/rolesAndStatus.ts

```diff
@@ -37,3 +37,10 @@ export const USER_PERMISSIONS = {
   TOPICS_READ: 'topics:read',
   METRICS_READ: 'metrics:read',
 };
+
+export const TOPIC_STATE = {
+  ACTIVE: 'Active',
```

Comment:
Although Figma states `Ready`, the BE returns `Active`. If it's decided to change the text to `Ready` then I'm sure we can quickly make that change on the BE.